tests/test_model_layers.py

```diff
@@ -513,6 +513,24 @@ def test_FunctionalChainInvolvementLink_attributes(
     link_uuid: str,
     target_uuid: str,
 ) -> None:
+    """
+    Test if a FunctionalChainLink covers all of its attributes.
+
+    Covered attributes are
+        * exchanged_items - List of all ExchangeItems spec. on the
+          FunctionalChainLink
+        * exchange_context - A context markup.
+
+    Parameters
+    ----------
+    fc_uuid
+        UUID of the FunctionalChain (i.e. parent of FunctionalChainLink)
+    link_uuid
+        UUID of the FunctionalChainLink
+    target_uuid
+        UUID of the FunctionalChainLink target (i.e. the element behind
+        `.involved`)
```

Comment:
I was confused what these UUIDs were about. Yes after less than a week. So I thought adding a docstring is a good thing.